tests/end2end/playwright/requests-api.spec.js

```diff
@@ -0,0 +1,170 @@
+            expect(json.rightsGroup["lizmap.tools.displayGetCapabilitiesLinks"].sort()).toStrictEqual(
+                [
+                    "admins",
+                    "anonymous",
+                    "group_a",
+                    "group_b",
+                    "Publishers"
+                ].sort()
+            );
+            expect(json.rightsGroup["lizmap.repositories.view"].sort()).toStrictEqual(
+                [
+                    "admins",
+                    "anonymous",
+                    "group_a",
+                    "group_b",
+                    "Intranet demos group",
+                    "lizadmins",
+                    "Publishers",
+                    "users"
+                ].sort()
+            );
+            expect(json.rightsGroup["lizmap.tools.loginFilteredLayers.override"].sort()).toStrictEqual(
+                [
+                    "admins",
+                    "Publishers"
+                ].sort()
+            );
+            expect(json.rightsGroup["lizmap.tools.layer.export"].sort()).toStrictEqual(
+                [
+                    "admins",
+                    "anonymous",
+                    "group_a",
+                    "group_b",
+                    "Publishers"
+                ].sort()
+            );
+            expect(json.rightsGroup["lizmap.tools.edition.use"].sort()).toStrictEqual(
+                [
+                    "admins",
+                    "anonymous",
+                    "group_a",
+                    "group_b",
+                    "Publishers"
+                ].sort()
+            );
+        });
+
+        test('GET all projects from a specific repository', async ({request}) => {
+            const response = await request.get(url + "/repositories/testsrepository/projects",
+                {
+                    headers: {
+                        authorization: 'Basic YWRtaW46YWRtaW4='
+                    }
+                });
+
+            const json = await checkJson(response);
+
+            expect(json[0].id).toBeDefined();
+            expect(json[0].title).toBeDefined();
+            expect(json[0].abstract).toBeDefined();
+        });
+
+        test('GET a specific project from a specific repository', async ({request}) => {
+            const response = await request.get(url + "/repositories/testsrepository/projects/attribute_table",
+                {
+                    headers: {
+                        authorization: 'Basic YWRtaW46YWRtaW4='
+                    }
+                });
+
+            const json = await checkJson(response);
+
+            expect(json.id).toBe("attribute_table");
+            expect(json.title).toBe("attribute_table");
+            expect(json.abstract).toBe("");
+            expect(json.keywordList).toBe("");
+            expect(json.proj).toBe("EPSG:2154");
+            expect(json.bbox).toBe("758745.34719999996013939, 6272204.95040000043809414, 782484.37399999995250255, 6286577.87310000043362379");
```

Comment:
Just to avoid a false positiv when the project `attribute_table` will be updated, can you change to a regepx ?

Something like `[\d+.\d+, ]+` ?

https://github.com/3liz/lizmap-web-client/blob/66892f696c84c587dd825098d8122806f69de1e7/tests/end2end/playwright/print.spec.js#L109